/*
The order in the list will determine the order of the project

"researchTitle" is the name of the project
"researchLead" is the leader of the project (likely one of the prinicipal investigators
"researchImage" is the link to the researchImage
"researchDescription" is the description of the project
"researchLinks" is the link to the publications
    Note, the publication link will be used to match with the publication information from the publication.js
*/
export const research = 
[
    {
        "researchTitle": "Plasma proteomics",
        "researchLead": "Maria Pernemalm",
        "researchImage": "",
        "researchDescription": "Being able to monitor these biomarkers by a simple blood test is in many cases the ultimate goal. The overall aim for the plasma proteomics team is to develop an accurate and sensitive pipeline for mass spectrometry based analysis of the plasma proteome. Due to the skewed protein concentration distribution and high patient to patient variability it has proved to be extremely difficult to detect protein biomarkers in plasma and many methods that are suitable for cellular and tissue proteomics performs poorly in plasma. Currently we are developing HiRIEF based methodologies both for global proteome analysis and targeted MRM-analysis of the plasma proteome. We are also taking advantage of the advances within the proteogenomics field and incorporating data on single amino acid variants in our analyses. Many of our studies aim to discover diagnostic, prognostic and predictive biomarkers in plasma, but we also have experience in other applications such as analysing the protein corona of nanoparticles or quantifying proteins from immunocapture analysis as well as analysing other biological fluids such as pleural effusion, cystic fluid, and cerebrospinal fluid.",
        "researchLinks": ["https://doi.org/10.1002/1878-0261.13355","https://doi.org/10.1186/s12931-023-02364-y","https://doi.org/10.3389/fendo.2022.971313"]
    },
    {
        "researchTitle": "Lung cancer",
        "researchLead": "Lukas Orre",
        "researchImage": "",
        "researchDescription": "Our general aim is to identify novel biomarkers and targets for cancer therapy to improve the treatment of non-small cell lung cancer (NSCLC). To accomplish this we are developing and applying advanced methods to study the effects of cancer therapy. We are specifically interested in therapies targeting receptor tyrosine kinases such as the epidermal growth factor receptor (EGFR). By determining the molecular response of cancer cells to different drugs we can identify the determinants of drug response or drug resistance, allowing us to suggest biomarkers and drug combinations for improved lung cancer therapy. The expression and functions of proteins in a cell determine its phenotype, i.e. the characteristics of the cell. For example the expression and functional activity of proteins in a cell will determine if a cell is cancerous or not, and whether it is sensitive to a specific type of treatment. Proteomics is the large-scale study of proteins in biological systems. The proteomics concept can be further developed into functional proteomics where the ambition is to also describe the functional state of proteins in large-scale experiments. For this purpose we are developing and applying proteomics methods that generate information about protein activity (e.g. phosphorylation), protein complex formation, and protein subcellular location. All of this data is then used in concert to understand why cells from different tumors respond differently to cancer drugs, and what combinations of cancer therapies we should select for each patient. Ultimately this knowledge will help us tailor the best therapy for each lung cancer patient.",
        "researchLinks": ["https://doi.org/10.1016/j.molcel.2018.11.035","https://doi.org/10.1038/s41596-022-00699-2","https://doi.org/10.1002/jev2.12128"]
    },
    {
        "researchTitle": "Breast cancer",
        "researchLead": "Henrik Johansson",
        "researchImage": "",
        "researchDescription": "Our overall aim in the BC team is to use MS based proteomics tools to increase our biological understanding of breast cancer, define clinical subtypes, and identify biomarkers for personalized therapy.<br/><br/>The estrogen and progesterone receptors are routinely used together with the tyrosine kinase receptor HER2 to stratify breast cancer patients into different clinical treatment regimes. Genomics efforts have further divided breast cancer, so called PAM50 subtypes, into Luminal A, B, HER2, Basal and normal-like subtypes based on mRNA expression, with refinement based on mutational patterns.<br/><br/>The drug tamoxifen has been one of the cornerstones of treatment of estrogen receptor positive breast cancer for nearly 4 decades. Tamoxifen treatment has reduced the recurrence rate by 50%, but about one-third of these patients still relapse. 80% of all breast cancer patients are eligible for tamoxifen, which highlights the need to find biomarkers to guide treatment. In our efforts to address this question, we have identified a 13 protein panel and the nuclear receptor, RARA, as potential predictive markers of tamoxifen resistance (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1038/s41467-019-09018-y' target='_blank'>Johansson, Nature Communications 2013; Johansson, Clinical Proteomics 2015</a>).<br/><br/>We use so called mass spectrometry (MS) based proteomics methods to characterize the proteome. Our method is based on isoelectric focusing of peptides followed by MS analysis that provide in-depth quantification of the proteome (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1038/nmeth.2732' target='_blank'>Branca et al Nature Methods 2014</a>). We use proteome information from breast tumors together with other levels of data as copy number, mRNA to gain novel understanding of BC biology. mRNA measurements is frequently used as surrogate measurements for protein and using our in-depth proteomics data we can elucidate the link between mRNA and protein. Since there is not always a direct correlation between mRNA and protein, the proteome provide a new avenue to identify biomarkers for personalized therapy. In addition to using protein levels as biomarkers, we are also investigating the use of protein interactions, as representing a functional unit, as biomarkers.<br/><br/>We are also interested in how the mutational landscapes affect the proteome and whether this information can be used as tumor specific antigens in immunotherapy.",
        "researchLinks": ["https://doi.org/10.1158/2767-9764.CRC-21-0100","https://doi.org/10.1038/s43018-021-00259-9","https://doi.org/10.21203/rs.3.pex-1565/v1"]
    },
    {
        "researchTitle": "Proteomics methods",
        "researchLead": "Rui Branca",
        "researchImage": "",
        "researchDescription": "In our group, we are developing and applying mass spectrometry (MS)-based methods with an aim to improve human proteome analysis. We have developed a prefractionation method known as high-resolution peptide isoelectric focusing (HiRIEF) (Branca, Nature Methods 2014). This method allows reproducible fractionation leading to a reduction of sample complexity prior to MS-analysis. This improves analytical depth, increases peptide and protein sequence coverage, improves overlap of detected peptides and proteins between samples, and provides an additional data point to each peptide, the peptide isoelectric point, allowing novel applications. One such novel application is using MS-data in an unbiased genome-wide search for protein coding DNA sequences in the human genome (proteogenomics) (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1038/nmeth.2732' target='_blank'>Branca, Nature Methods 2014; Boekel Nature Biotechnology 2015</a>). We are also developing methods to improve tissue and plasma proteome analysis, detection and quantification of post-translational modifications (most notably phosphorylations, glycosylations, and redox related cysteine modifications), and targeted analysis of proteins. To interpret the vast amounts of information generated by proteomics technologies, we are actively working on development and use of bioinformatics tools. We have recently developed novel tools for splice variant specific quantification at protein level (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1038/s41467-018-03311-y' target='_blank'>Zhu, Mol. Cell. Prot. 2014</a>), a galaxy based proteogenomics pipeline (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1038/nbt.3134' target='_blank'>Boekel, Nature Biotechnology</a>), and methods to improve quantification accuracy in proteomics (<a className='cursor-pointer' class='textLink' href='https://doi.org/10.1007/978-1-4614-7209-4_9' target='_blank'>Forshed, Mol. Cell. Prot. 2011</a>; <a className='cursor-pointer' class='textLink' href='https://doi.org/10.1074/mcp.m112.021592' target='_blank'>Hultin-Rosenberg Mol. Cell. Prot. 2013</a>; <a className='cursor-pointer' class='textLink' href='https://doi.org/10.1016/j.jprot.2013.10.035' target='_blank'>Sandberg J. Proteomics 2014</a>).",
        "researchLinks": ["https://doi.org/10.1038/s41467-019-09018-y","https://doi.org/10.1038/s41467-018-03311-y","https://doi.org/10.7554/eLife.71156"]
    },
    {
        "researchTitle": "Childhood cancer",
        "researchLead": "Rozbeh Jafari",
        "researchImage": "",
        "researchDescription": "Our overall aim in the Childhood Cancer team is to use mass spectrometry based proteomics to increase our biological understanding of Acute Lymphoblastic Leukemia (ALL) and Neuroblastoma, define clinical subtypes, evaluate drug interactions and identify biomarkers for targeted thereapies, improving personalized therapy.<br/><br/>Childhood ALL is the most common pediatric cancer accounting for roughly 75-80% of all childhood cancer cases. Although many of the patients achieve complete remission and are cured by standard therapies, patientsâ€™ relapsing in the disease have a poor prognosis. Treatment of relapsed ALL is overall more intense and is affecting the quality of life considerably. Therfore the resistance development and long-term side effects of current broad cytotoxic therapy of ALL has warranted novel predictive biomarkers as well as novel targeted drug therapeutics.<br/><br/>We are engaged in using Thermal Proteome Profiling and in-depth proteogenomics for identifying novel predictive resistant and therapeutic biomarkers for a more effective personalized therapy in treatment of pediatric ALL.",
        "researchLinks": ["https://doi.org/10.1038/s43018-021-00259-9","https://doi.org/10.1038/s41467-022-29224-5","https://doi.org/10.1038/s41467-022-33385-8"]
    },
    {
        "researchTitle": "Molecular Tumor Board Portal",
        "researchLead": "David Tamborero",
        "researchImage": "",
        "researchDescription": "Our objective is to support the clinical decision-making based on the interpretation of the omics data from the patient tumors. Precision oncology relies on the accurate discovery and interpretation of tumor alterations, and vast efforts are ongoing to better understand their significance for the diagnosis, prognosis and therapy selection of the disease. However, the knowledge continuously generated by the community is scattered across a multitude of resources that follow disparate data models, which impedes the use of this information in the clinical setting. Moreover, most of the variants that are observed in a tumor are of unknown significance and additional computational tools are required to estimate their relevance.<br/><br/>To address these challenges, we have developed the Molecular Tumor Board (MTB) portal, a platform for the annotation and distribution of tumor omics data. The MTB portal employs state-of-the-art knowledge bases and bioinformatics tools to interpret the biological and clinical significance of the tumor alterations according to distinct levels of evidence. On the one hand, the use of the portal guarantees that clinical decisions such as the inclusion of the patient in a clinical trial, the off-label use of anti-cancer drugs or the need for genomic counseling are based on the most updated and comprehensive knowledge that is currently available. On the other, the use of a centralized framework to annotate and collect the results of these efforts will provide the resource to generate novel knowledge and better guide these same decisions in the patients of the future.<br/><br/>The MTB portal is currently used by the seven leading european oncology centers forming the <a className='cursor-pointer' class='textLink' href='https://cancercoreeurope.eu/' target='_blank'>Cancer Core Europe</a>. At present, the MTB portal supports genomic data analyses, but we aim to implement interpretation of other omics data types as transcriptomics, proteomics and ex-vivo drug testing results.",
        "researchLinks": ["https://doi.org/10.1038/s41591-020-0969-2","https://doi.org/10.1038/s43018-022-00332-x"]
    }
]